js_modules/dagster-ui/packages/ui-core/src/launchpad/ConfigEditorConfigPicker.tsx

```diff
@@ -280,7 +280,6 @@ const ConfigEditorPartitionPicker: React.FC<ConfigEditorPartitionPickerProps> =
     return (
```

Comment:
I don't think the key was necessary here, I removed it because I use the key '1' or '0' below in `CreatePartitionDialog` to reset its state which collides with a partition if its name is '1' or '0'.

What was the original intent here? @bengotow. Everything seems to work fine without it.

Reply:
leaving it uncontrolled is fine, as long as theres no other way to set the partition name elsewhere (is there?)? Maybe its the case where you switch between tabs and the second tab has a different partition and this component doesn't update? If that's the case I guess I can add it back and just use a different key below (or move this key high up the component tree so that the tree doesn't reuse the state from the old tree (might be better tbh, less possibility of bugs)).